out_no_browser.json

```diff
@@ -0,0 +1 @@
{"hasAvailability":true,"numberAvailableAppointments":1,"signUpLink":"https://www.maimmunizations.org/client/registration?clinic_id=3205"},"03/31/2021":{"hasAvailability":true,"numberAvailableAppointments":2,"signUpLink":"https://www.maimmunizations.org/client/registration?clinic_id=3264"}},"hasAvailability":true,"extraData":{"Vaccinations offered":"Pfizer-BioNTech COVID-19 Vaccine","Age groups served":"Seniors, Other, Adults","Services offered":"Vaccination","Additional Information":"By Appointment Only - Arrive 15min before scheduled appointment. Do not enter the building until 5 min before your scheduled appt time. Watch the digital scoreboard for appointment time prompting. BRING PHOTO ID AND INSURANCE CARD(S). MASKS are required and must be worn properly at all times. If receiving your 2nd dose, please bring your vaccine card with you. Wear clothing that allows a clinician to easily access your upper arm. The vaccine is delivered to the deltoid muscle, the big muscle on your shoulder. Consider wearing a short-sleeved shirt, or wear a short-sleeved shirt under a sweater or jacket that can be easily removed.","Clinic Hours":"01:30 pm - 05:30 pm"},"timestamp":"2021-03-29T15:46:00.926Z","latitude":42.2649739,"longitude":-71.7965785},{"name":"Cape Cod Community College","street":"2240 Iyannough Road","city":"West Barnstable","zip":"02668","availability":{"03/30/2021":{"hasAvailability":true,"numberAvailableAppointments":5,"signUpLink":null},"03/31/2021":{"hasAvailability":true,"numberAvailableAppointments":8,"signUpLink":"https://www.maimmunizations.org/client/registration?clinic_id=3923"}},"hasAvailability":true,"extraData":{"Vaccinations offered":"Pfizer-BioNTech COVID-19 Vaccine","Age groups served":"Adults","Services offered":"Vaccination","Additional Information":"Pfizer vaccine. All those eligible in Phase 1 and people who are 60 or older, people with 2 or more certain medical conditions, people who live or work in low income and affordable senior housing, K-12 educators, K-12 school staff, and child care workers, and certain workers","Clinic Hours":"10:00 am - 04:00 pm"},"timestamp":"2021-03-29T15:46:00.944Z","latitude":41.6909399,"longitude":-70.3373802},{"name":"Barnstable County Fairgrounds","street":"1220 Nathan Ellis Hwy","city":"East Falmouth","zip":"02630","availability":{"04/01/2021":{"hasAvailability":true,"numberAvailableAppointments":1,"signUpLink":null}},"hasAvailability":true,"extraData":{"Vaccinations offered":"Pfizer-BioNTech COVID-19 Vaccine","Age groups served":"Adults","Services offered":"Vaccination","Additional Information":"All those eligible in Phase 1 and people who are 60 or older, people with 2 or more certain medical conditions, people who live or work in low income and affordable senior housing, K-12 educators, K-12 school staff, and child care workers, and certain workers. Pfizer vaccine","Clinic Hours":"09:45 am - 04:00 pm"},"timestamp":"2021-03-29T15:46:01.468Z","latitude":41.6181548,"longitude":-70.5328427}]}
```

Comment:
This file should not be included.  I just added it to `.gitignore` in the PR #193

Reply:
this was an accident, thanks